packages/ml/examples/Advanced/ml_user_aggr.cpp

```diff
@@ -48,7 +48,7 @@ using namespace Galeri;
 
 char*UserLabel()
 {
-  return("Linear");
+   return((char*)"Linear");
```

Comment:
The original code was broken with respect to build warnings (it's Ungood to cast a constant string to nonconst) so I think this is OK.

Reply:
Thanks Mark! I think it's ugly, but as you say, ok.